dips/dip-158.md

```diff
@@ -65,59 +59,43 @@ Content:
+7. The wallet initializes the payment protocol (off-chain) which will end (if successful) with the funds being charged. Successful completion implies that the receiving VASP was satisfied with the payer information provided and the sender VASP was satisfied with the merchant information provided. **Note** that if the payment amount exceeds the amount that requires a dual attestation signature to be submitted to the blockchain, it must be part of the on-chain transaction submitted  
 
 ### **Charge Payment Type** 
 Charge payment stipulates an immediate capture of funds. This means that a successful off-chain negotiation for such payment will result in funds being captured.
-
 As soon as the funds are captured, i.e. the sender agreed to the payment terms, the sender VASP is expected to submit a corresponding transaction on-chain and actually transfer the funds. 
 
-
-### **Auth / Capture Payment Type**
-Auth / Capture payment, as the name suggests, means that the payment is divided into two steps:
-* Authorization
-* Capture
- 
-The Authorization step ends when the funds are locked by the sender VASP so they can later be captured by the receiver VASP. Note that the capture may occur several days after a successful authorization. It is up to the receiver VASP to decide on the timing. In addition, the receiver may choose to capture only a partial amount rather than the full amount which was authorized. 
-
-This payment type is useful for delayed fulfillment or pre-authorizing an expected amount to ensure that an amount can be captured after services are rendered. 
-
-Off-chain negotiations for Auth / Capture payment are divided into two steps accordingly. First, off-chain negotiations will result in funds being authorized, i.e. locked by the wallet. Later, a separate off-chain negotiation triggered by the receiving VASP will capture the funds. Note that for the time being, multi-capture is not supported and there can be only a single capture request.
-
-Successful authorization will not necessarily be followed by a successful capture. It is possible that the receiver will choose to cancel the authorization and essentially void the payment before it was captured. One possible reason to void a payment is that the products for which an amount was locked are out of stock and cannot be fulfilled.
+### **Other Payment Types (future)**
+More advanced eCommerce payments flows will require additional payment types which will be be defined in future DIPs.
```

Comment:
perhaps replace "more advanced" with "more arcane"? JK. More seriously, perhaps replace with "Protocols supporting other eCommerce payment flow ..."

Reply:
Though arcane is more accurate, I went with your latter suggestion :))
Updated.